Documentation/data-collection.md

```diff
@@ -134,6 +134,8 @@ data:
     # subscription_sync_total is the number of times an OLM operator
     # Subscription has been synced, labelled by name and installed csv
     - '{__name__="subscription_sync_total"}'
+    # olm_resolution_duration_seconds is the duration of a dependency resolution attempt
```

Comment:
nit: i believe this is missing an owning team. I assume this is `@openshift/openshift-team-olm`?

Reply:
I was unaware that the OLM team had a tag, where was @openshift/openshift-team-olm generated ?

Reply:
Added in the latest version of the release.